js/exporter/jspdf/v3/pdf_utils_v3.js

```diff
@@ -68,21 +74,21 @@ function getLineHeightShift(doc) {
     return (doc.getLineHeightFactor() - DEFAULT_LINE_HEIGHT) * doc.getFontSize();
 }
 
-function drawTextInRect(doc, text, rect, verticalAlign, wordWrapEnabled, jsPdfTextOptions) {
-    const textArray = getTextLines(doc, text, doc.getFont(), { wordWrapEnabled, columnWidth: rect.w });
+function drawTextInRect(doc, text, textRect, verticalAlign, wordWrapEnabled, jsPdfTextOptions) {
```

Comment:
imho it's obvious that 'rect' is the 'target rect to paint the text'
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-drawtext - 'rect'
https://developer.apple.com/documentation/uikit/uilabel/1620527-drawtext - 'rect'
It can be 'targetRect'
Why we need to rename it to 'textRect'?